gdk4/Gir.toml

```diff
@@ -71,6 +71,10 @@ generate = [
     "Gdk.VulkanError",
 ]
 
+ignore = [
+    "Gdk.DrawingContext", # not useful
```

Comment:
Not? Why not? :)

Reply:
I couldn't find anything that uses it and it's only defined in sys/lib.rs as 
```
#[repr(C)]
pub struct _GdkDrawingContext(c_void);

pub type GdkDrawingContext = *mut _GdkDrawingContext;
```

Reply:
Indeed seems completely unused. Strange. Please create an upstream issue then :)

Reply:
Especially that there's a `GdkDrawContext`

Reply:
Fixed in https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/3232